### DISCUSSION TITLE
pfSense Elastic Dashboard Empty?

### DISCUSSION THREAD

**Discussion by sjnhawk** [Security-Onion-Solutions/securityonion#11488] [Unsupported Versions] [answered]
Looking at the asset dashboards for the pfSense integration in Kibana, and mostly everything is empty. Is that expected? I know I have logging properly configured in pfSense and am receiving the logs. Am I missing something here?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/67609108/1e7c7a85-4df7-4dfc-87f6-5947868e1f4f)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/67609108/3a9930ee-a41c-4ced-8d89-5f57a35a8092)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/67609108/31a6cc97-6f76-49d9-a7b6-1da565f7cb45)

**Comment by sjnhawk** [comment]
*Of note, the firewall selector dropdown only provides me with the hostname of my SO instance, and nothing shows up under the interface selectors.

**Comment by TOoSmOotH** [comment, marked as answer]
We modify the fields so you can pivot in hunt from the logs to other logs. There is a firewall dashboard inside of SOC. If you want to use Kibana for this you would need to modify the visualizations to point to the appropriate fields.
